scripts/busybox.sh

```diff
@@ -47,6 +47,13 @@ while true; do
 done
 
 :	${usrdir:="${PWD}"/usr}
+
```

Comment:
Actually there is no point for this... the installed files should have the right path. This a matter of setting up the right in the Makefile. -- I will do it in a commit.